datamade/committee-oversight
# Category detail page

## Overview

Connects #67. This PR creates a Wagtail Page for automatically populated and dynamically updated Category detail pages. No CSS work yet but the elements are all there! I've left in-line notes below.


## Testing Instructions

* Delete your existing volume (not positive you'll have to do this, but I did): 
```
docker-compose down
docker volume rm committee-oversight_committeeoversight-db-data
``` 
 * Restart the app: 
```
docker-compose up
```
 * Open a new terminal tab. Load in the sample data:
```
docker-compose exec postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump
```
 * Apply new migrations: 
```
docker-compose run --rm app python manage.py migrate
```
 * Navigate to http://localhost:8000/cms/pages/2/ and click "ADD CHILD PAGE"
*  Select "Category detail page"
* Create a new category page and make sure everything's intuitive. Save it as a draft; publish it; add images and text to the "body" field. Delete it. 
* Try different categories and make sure the hearings table is populating correctly (the hearings in `hearings.dump` don't have categories; you'll have to go to http://localhost:8000/list to edit them)
